app/assets/stylesheets/application.scss

```diff
@@ -12,6 +12,8 @@ $govuk-include-default-font-face: false;
 // Components from govuk_publishing_components gem
 @import "govuk_publishing_components/govuk_frontend_support";
 
+@import "components/intervention";
```

Comment:
We can remove this import if the approach mentioned in `application.html.erb` is used